pkg/proc/variables_test.go

```diff
@@ -1575,6 +1575,57 @@ func testCallFunctionIntl(t *testing.T, grp *proc.TargetGroup, p *proc.Target, t
 	}
 }
 
+func TestIssue4051(t *testing.T) {
+	protest.AllowRecording(t)
+	withTestProcess("issue4051", t, func(p *proc.Target, grp *proc.TargetGroup, fixture protest.Fixture) {
+		err := grp.Continue()
+		assertNoError(err, t, "initial continue to breakpoint failed")
+
+		err = proc.EvalExpressionWithCalls(grp, p.SelectedGoroutine(), `main.Hello("world")`, pnormalLoadConfig, true)
```

Comment:
This fails on linux/386 because we don't support function calls there and the error is different. I think we can just add a `protest.MustSupportFunctionCalls` at the beginning.

Reply:
Done.